openedx/core/djangoapps/appsembler/settings/settings/common.py

```diff
@@ -20,6 +20,7 @@ def plugin_settings(settings):
         'openedx.core.djangoapps.appsembler.sites',
         'openedx.core.djangoapps.appsembler.html_certificates',
         'openedx.core.djangoapps.appsembler.api',
+        'openedx.core.djangoapps.appsembler.honeycomb',  # legacy honeycomb middleware
     )
 
     settings.DEFAULT_TEMPLATE_ENGINE['OPTIONS']['context_processors'] += (
```

Comment:
```suggestion
    settings.MIDDLEWARE_CLASSES += [
        'openedx.core.djangoapps.appsembler.honeycomb.middleware.HoneycombLegacyMiddleware'
    ]

    settings.DEFAULT_TEMPLATE_ENGINE['OPTIONS']['context_processors'] += (
```

here or just in the `aws_common.py`:

https://github.com/appsembler/edx-platform/blob/1d4d8005c87e567f6985a3835373c4819d8e6afe/openedx/core/djangoapps/appsembler/settings/settings/aws_common.py#L67-L69